.github/workflows/python_release.yml

```diff
@@ -0,0 +1,100 @@
+name: Reusable python Release workflow example
+
+on:
+  workflow_call:
+    secrets:
+      GCP_PROJECT_ID:
+        required: true
+      GCP_ARTIFACT_REGISTRY_URI:
+        required: true
+      GCP_DOCKER_REGISTRY:
+        required: true
+      GCP_SA_KEY:
+        required: true
+      GCP_ARTIFACT_REGISTRY_PYTHON_URI:
+        required: true
+
+
+      
+    inputs:
+      runs-on:
+        description: Platform to execute on
+        required: false
+        type: string
+        default: ubuntu-latest
+  
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.9]
+
+    env:
+      PROJECT_ID: ${{ secrets.GCP_PROJECT_ID }}
+      ARTIFACT_REGISTRY: ${{ secrets.GCP_ARTIFACT_REGISTRY_URI }}
+      DOCKER_REGISTRY: ${{ secrets.GCP_DOCKER_REGISTRY }}
+      SA_KEY: ${{ secrets.GCP_SA_KEY }}
+      GIT_TERMINAL_PROMPT: 0
```

Comment:
What does this do?

Reply:
If I remember correctly this might be related to running the workflow offline. There is a bunch of magic setup in the github runners that disables interactive "actions".